lib/amazonka-core/test/Test/Amazonka/Sign/V4/Base.hs

```diff
@@ -0,0 +1,115 @@
+-- |
+-- Module      : Test.AWS.Sign.V$
```

Comment:
```suggestion
-- Module      : Test.AWS.Sign.V4
```

Reply:
Hah, thanks. I might have fat fingered that.